perimysium/dataman.py

```diff
@@ -409,6 +409,18 @@ def write(self, fpath):
 
         f.close()
 
+    def addnoise(self, noiseWidth):
```

Comment:
I initially thought that a user may not want to overwrite the original object, rather clone and do the operations.